src/Build/Setup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Could you apply same changes [here](https://github.com/amaneureka/AtomOS/tree/master/libs) too